.github/configs-docker/multi-config.toml

```diff
@@ -1 +1,13 @@
-# Produces multiple artifacts, in nested paths.
\ No newline at end of file
+# Produces multiple artifacts, in nested paths.
+command = [
+  "touch",
+  "one.multi",
+  "&&",
```

Comment:
how would this work? I don't believe there is a shell involved (but maybe I am missing something)

Reply:
(actually maybe there is, I see bash mentioned in the Docker command)

Reply:
we should probably update it to invoke "/bin/sh -c"

Reply:
perhaps we can spin the wheel of fortune once more with https://github.com/slsa-framework/example-package/pull/347. 

I can investigate more tomorrow also.